sdk/resourcemanager/azure-resourcemanager-network/CHANGELOG.md

```diff
@@ -2,14 +2,17 @@
 
 ## 2.51.0-beta.1 (Unreleased)
 
-### Features Added
-
 ### Breaking Changes
 
-### Bugs Fixed
+- The details that query used to fetch are now ingested directly in Customer's log Analytics workspace and hence query operation is no more valid on connection Monitor resource now.
```

Comment:
```suggestion
- Removed `query` from `ConnectionMonitor`. Its functionality is now provided by Log Analytics service.
```

Reply:
Fixed by [commit#877beac](https://github.com/Azure/azure-sdk-for-java/pull/45455/commits/877beac9261417b2b8748ba5f780045e6351671a)